sympy/core/tests/test_wester.py

```diff
@@ -1223,7 +1223,7 @@ def test_N12():
 
 def test_N13():
     x = Symbol('x')
-    assert solveset(sin(x) < 2, domain=S.Reals) == S.Reals
+    assert solveset(sin(x) < 2, domain=S.Reals) == Interval(0, 2*pi, False, True)
```

Comment:
> Ans should be all +ve real numbers, isn't it ?

Yes. You are mathematically correct.
I have stated the reason for doing so in my blog [post](http://kshitij10496.github.io/2016/08/07/GSoC-Week-10-11/).

Restricting the domain of solution now, will help us later in extension (with the implementation of `BigUnion`).